src/tests/dictionaryConcepts/container/DictionaryMappings.test.jsx

```diff
@@ -0,0 +1,241 @@
+          collectionName: 'dev-col',
+          dictionaryName: 'dev-col',
+        },
+      },
+      location: {
+        pathname: '/random/path',
+      },
+      fetchDictionaryMappings: jest.fn(),
+      mappings: [
+        {
+        to_concept_name: 'random_name',
+        from_concept_name: 'random_name',
+        map_type: 'Map As Type',
+        source: 'random source',
+      },
+        {
+        to_concept_name: 'random_name',
+        from_concept_name: 'random_name',
+        map_type: 'Map As Type',
+        source: 'random source',
+      },
+        {
+        to_concept_name: 'random_name',
+        from_concept_name: 'random_name',
+        map_type: 'Map As Type',
+        source: 'random source',
+      },
+        {
+        to_concept_name: 'random_name',
+        from_concept_name: 'random_name',
+        map_type: 'Map As Type',
+        source: 'random source',
+      },
+        {
+        to_concept_name: 'random_name',
+        from_concept_name: 'random_name',
+        map_type: 'Map As Type',
+        source: 'random source',
+      },
+        {
+        to_concept_name: 'random_name',
+        from_concept_name: 'random_name',
+        map_type: 'Map As Type',
+        source: 'random source',
+      },
+        {
+        to_concept_name: 'random_name',
+        from_concept_name: 'random_name',
+        map_type: 'Map As Type',
+        source: 'random source',
+      },
+        {
+        to_concept_name: 'random_name',
+        from_concept_name: 'random_name',
+        map_type: 'Map As Type',
+        source: 'random source',
+      },
+        {
+        to_concept_name: 'random_name',
+        from_concept_name: 'random_name',
+        map_type: 'Map As Type',
+        source: 'random source',
+      },
+        {
+        to_concept_name: 'random_name',
+        from_concept_name: 'random_name',
+        map_type: 'Map As Type',
+        source: 'random source',
+      },
+        {
+        to_concept_name: 'random_name',
+        from_concept_name: 'random_name',
+        map_type: 'Map As Type',
+        source: 'random source',
+      },
+      ],
+      filteredClass: ['Diagnosis'],
+      filteredSources: ['CIEL'],
+      loading: false,
+      to: 'random',
+    };
+    localStorage.setItem('dictionaryPathName', '/concepts/users/admin/858738987555379984/Malaria/en');
+    wrapper = mount(<Provider store={store}>
+      <Router>
+        <DictionaryMappings {...props} />
+      </Router>
+    </Provider>);
+    expect(wrapper.find('.-totalPages').text()).toBe('2');
+  });
+
+  it('should test componentWillReceiveProps', () => {
+    const app = shallow(<DictionaryMappings {...props} />);
+    const newProps = {
+      mappings: [{
+        to_concept_name: 'random_name',
+        from_concept_name: 'random_name',
+        map_type: 'Map As Type',
+        source: 'random source',
+      }],
+    };
+
+    app.setState({ searchInput: 'ciel' });
+    app.setProps(newProps);
```

Comment:
@dessHub , you need to pass an `expect` statement to make the comparison

Reply:
Fixed it. I didn't actually need this edge case.